tests/codegen/asm-clobber_abi-avr.rs

```diff
@@ -0,0 +1,43 @@
+//@ assembly-output: emit-asm
```

Comment:
Could you move this file into tests/codegen/asm directory like https://github.com/rust-lang/rust/pull/132456?

Reply:
Done :+1: